python/hopsworks/core/environment_api.py

```diff
@@ -29,7 +32,7 @@ def __init__(
 
         self._environment_engine = environment_engine.EnvironmentEngine(project_id)
 
-    def create_environment(self, await_creation=True):
+    def create_environment(self, name: str, description: Optional[str] = None, base_environment_name: Optional[str] = "python-feature-pipeline", await_creation: Optional[bool] = True) -> environment.Environment:
```

Comment:
If the `base_environment_name` is the name of the image used as base for this environment, we can provide an Enum class to help users pick the correct names for the base images.

Reply:
The backend validates the name and sends a list of valid environment names back. I think adding the enum will come at a later stage maybe when I'm back.